Src/FluentAssertions/Numeric/NumericAssertions.cs

```diff
@@ -481,5 +481,27 @@ public AndConstraint<TAssertions> NotBeOfType(Type unexpectedType, string becaus
             throw new NotSupportedException("Calling Equals on Assertion classes is not supported.");
 
         private protected virtual bool IsNaN(T value) => false;
+
+        /// <summary>
+        /// A method to generate additional information upon comparison failures.
+        /// </summary>
+        /// <param name="expected">The value to compare the current numeric value with.</param>
+        /// <returns>
+        /// Returns the difference between the integral number value and the <paramref name="expected" /> value.
```

Comment:
This method is not only for integral values, floats, doubles and decimals are e.g. also covered.